src/tactic/ext.lean

```diff
@@ -150,6 +150,64 @@ do e  ← saturate_fun n,
    e' ← saturate_fun n',
    unify e e' <|> fail format!"{n} and {n'} are not definitionally equal types"
 
+section performance_hack
+/-!
+For performance reasons, it is inadvisable to use `user_attribute.get_param`.
```

Comment:
Should we add this as a library note, and/or to the doc string of `get_param` in core? This isn't the first time we've seen performance problems with people accidentally using it.

Reply:
I've added a library note.